ArduCopter/AP_Arming.cpp

```diff
@@ -552,7 +557,11 @@ bool AP_Arming_Copter::mandatory_gps_checks(bool display_failure)
             return false;
         }
     }
-
+    // check for copter velocity innovations
+    if (!vel_innovation_check()) {
```

Comment:
```suggestion
    if (mode_request_gps && !vel_innovation_check()) {
```

If someone wants to take off in stabilise with a bad velocity - let them.

Frankly I'm really not sure we should make it a mandatory check.  Are we going to save more vehicles from tipping over?  Sure!  If we never let them arm at all we'll save every vehicle....  How many users are we going to frustrate with this check vs vehicles saved, I wonder.